Session/Database/Convenience/Interaction+UI.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import UIKit
+import SessionUIKit
+import SessionMessagingKit
+
+public extension Interaction.State {
+    func statusIconInfo(
+        variant: Interaction.Variant,
+        hasBeenReadByRecipient: Bool,
+        hasAttachments: Bool
+    ) -> (image: UIImage?, text: String?, themeTintColor: ThemeValue) {
+        guard variant == .standardOutgoing else {
+            return (nil, nil, .messageBubble_deliveryStatus)
+        }
+
+        switch (self, hasBeenReadByRecipient, hasAttachments) {
+            case (.deleted, _, _), (.localOnly, _, _):
+                return (nil, nil, .messageBubble_deliveryStatus)
+            
+            case (.sending, _, true):
+                return (
+                    UIImage(systemName: "ellipsis.circle"),
```

Comment:
Do we want to change these to use Lucide icons?

Reply:
Yea we should swap these to use Lucide icons but we should probably do it in a separate PR as part of a larger icon refresh